data/core/encyclopedia/geography.cfg

```diff
@@ -30,7 +30,7 @@
 [topic]
     id=great_river
     title= _ "Great River"
-    text= _ "The Great River forms the boundary between the <ref>dst='kingdom_wesnoth' text='Kingdom of Wesnoth'</ref> and the <ref>dst='northlands' text='Northlands'</ref>. It empties to the <ref>dst='great_ocean' text='ocean'</ref> at <ref>dst='elensefar' text='Elensefar'</ref>."
+    text= _ "The Great River forms the boundary between the <ref>dst='kingdom_wesnoth' text='Kingdom of Wesnoth'</ref> and the <ref>dst='northlands' text='Northlands'</ref>. It empties to the <ref>dst='great_ocean' text='ocean'</ref> at <ref>dst='elensefar' text='Elensefar'</ref>. The Ford of Abez is used a crossing between Wesnoth and the Northlands, and is the westernmost point where the Great River can be crossed without ships."
```

Comment:
"is used a crossing" sounds like two different phrasings got mashed together. It could be "is a crossing" or "is used as a crossing" or maybe "is used to cross", though that last one doesn't seem so great.